Misc/NEWS.d/next/Library/2024-06-15-23-38-36.gh-issue-120284.HwsAtY.rst

```diff
@@ -0,0 +1,4 @@
+Enhance :meth:`asyncio.Runner.run` to accept the same object types as
+:meth:`asyncio.BaseEventLoop.run_until_complete`. Instead of accepting
+only a coroutine, it now accepts an awaitable, coroutine,
```

Comment:
```suggestion
only a :term:`coroutine`, it now accepts an :term:`awaitable`, :term:`coroutine`,
```

Reply:
I'd be happy to make this change, but I'm still running into an issue with the reference to asyncio.BaseEventLoop.run_until_complete not being accepted:

`/home/runner/work/cpython/cpython/build/NEWS:183: WARNING: py:meth reference target not found: asyncio.BaseEventLoop.run_until_complete`

Any suggestions for fixing that?

Reply:
Fixed it

Reply:
Thanks - I have also added the change requested above as commit df87f52.